### PR TITLE
SNOW-733699 make Parquet the default blob format version instead of Arrow

### PR DESCRIPTION
**Pull request by sfc-gh-gdoci**
This PR switches the default blob format version from Arrow to Parquet.

### CODE REVIEW

**Review by sfc-gh-azagrebin — COMMENTED** [review]
Thanks Gloria!
I think `System.getProperty("enableParquetTests")` should be also removed to run both arrow and parquet.

**Review by sfc-gh-azagrebin — APPROVED** [review]

File: src/test/java/net/snowflake/ingest/streaming/internal/SnowflakeStreamingIngestChannelTest.java [hunk 1]
```diff
@@ -463,9 +462,9 @@ public void testOpenChannelSuccessResponse() throws Exception {
 
   @Test
   public void testInsertRow() {
-    SnowflakeStreamingIngestClientInternal<VectorSchemaRoot> client =
+    SnowflakeStreamingIngestClientInternal<ParquetChunkData> client =
```
Comment (sfc-gh-tzhang) [inline src/test/java/net/snowflake/ingest/streaming/internal/SnowflakeStreamingIngestChannelTest.java:465]: would this still work with Arrow?

**Review by sfc-gh-tzhang — APPROVED** [review]
Thanks @sfc-gh-gdoci ! Could you make sure everything still work with Arrow if we set the version to one in the config file? In case there is an issue, customer could mitigate it by switching back.

File: src/test/java/net/snowflake/ingest/streaming/internal/SnowflakeStreamingIngestChannelTest.java (hunk 1 above)
Comment (sfc-gh-gdoci) [reply src/test/java/net/snowflake/ingest/streaming/internal/SnowflakeStreamingIngestChannelTest.java]: It wouldn't. So I changed this test. Other tests should not need any modification and work with both Arrow/Parquet as default versions.

File: src/test/java/net/snowflake/ingest/streaming/internal/SnowflakeStreamingIngestChannelTest.java [hunk 2]
```diff
@@ -463,9 +463,16 @@ public void testOpenChannelSuccessResponse() throws Exception {
...
-        new SnowflakeStreamingIngestClientInternal<>("client");
-    SnowflakeStreamingIngestChannelInternal<VectorSchemaRoot> channel =
+    SnowflakeStreamingIngestClientInternal<?> client;
+    boolean isArrowDefault =
+        ParameterProvider.BLOB_FORMAT_VERSION_DEFAULT == Constants.BdecVersion.ONE;
+    if (isArrowDefault) {
+      client = new SnowflakeStreamingIngestClientInternal<VectorSchemaRoot>("client_ARROW");
+    } else {
+      client = new SnowflakeStreamingIngestClientInternal<ParquetChunkData>("client_PARQUET");
+    }
```
Comment (sfc-gh-tjones) [inline src/test/java/net/snowflake/ingest/streaming/internal/SnowflakeStreamingIngestChannelTest.java:473]: Nit, can consolidate to:
```
SnowflakeStreamingIngestClientInternal<?> client = ParameterProvider.BLOB_FORMAT_VERSION_DEFAULT == Constants.BdecVersion.ONE ?
    new SnowflakeStreamingIngestClientInternal<VectorSchemaRoot>("client_ARROW") :
    new SnowflakeStreamingIngestClientInternal<ParquetChunkData>("client_PARQUET");
```

**Review by sfc-gh-tjones — APPROVED** [review]

**Review by sfc-gh-tjones — APPROVED** [review]

**Pull request merged**